app.js

```diff
@@ -13,6 +13,49 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function stateToUrl(state) {
+  var qsParams = Object.keys(state);
+
+  if (!qsParams.length) {
+    return "";
+  }
+
+  return "?" + qsParams.map(function(param) {
```

Comment:
Considering we already use it in the code, I assume it's not a problem: 
https://github.com/captainbrosset/devtools-bug-finder/blob/gh-pages/app.js#L17